drivers/regulator/Kconfig.npm1300

```diff
@@ -6,23 +6,22 @@ config REGULATOR_NPM1300
 	default y
 	depends on DT_HAS_NORDIC_NPM1300_REGULATOR_ENABLED
 	select I2C
+	select MFD
 	help
 	  Enable the Nordic nPM1300 PMIC regulator driver
 
-if REGULATOR_NPM1300
-
 config REGULATOR_NPM1300_COMMON_INIT_PRIORITY
 	int "nPM1300 regulator driver init priority (common part)"
 	default 75
+	depends on REGULATOR_NPM1300
 	help
 	  Init priority for the Nordic nPM1300 regulator driver (common part).
 	  It must be greater than I2C init priority.
 
 config REGULATOR_NPM1300_INIT_PRIORITY
 	int "nPM1300 regulator driver init priority"
 	default 76
+	depends on REGULATOR_NPM1300
 	help
 	  Init priority for the Nordic nPM1300 regulator driver. It must be
 	  greater than REGULATOR_NPM1300_COMMON_INIT_PRIORITY.
-
-endif
```

Comment:
the guidelines here is to use `if` when the number of options is > 1, `depends on` for single options